### ISSUE TITLE
Edit ganache-cli launch parameters, enigma-contract docker image

### ISSUE THREAD

**Issue by levackt** [enigmampc/dev-help#14]
https://forum.enigma.co/t/edit-ganache-cli-launch-parameters-enigma-contract-docker-image/1021

joao.l.martins would like to provide his own mnemonic key, ideally without building an image.
He's built a custom image in the meantime but couldn't run it.

There's also a follow up question, how secret contracts call smart contracts with computation results.

**Comment by ainsleys** [comment]
Hey @levackt -- thanks for adding this one! Let's use this issue to address factors around the mneumonic key without building an image. I don't have an answer for that right now... I'll open a new issue for his second question so we can keep things organized/searchable here.

**Comment by lacabra** [comment]
Yes, we can implement this easily with the use of environment variables. This is an area of active work as we prepare for testnet and mainnet, and it's a feature of how we will use system wide tests to configure a multitude of parameters using environment variables (that get passed onto the containers), instead of command line arguments. I cannot give you an exact timing of the implementation of this feature, but it's planned.

**Comment by ainsleys** [comment]
Added feature request future tag to show we will implement this. Closing the issue as we cannot do it now.

**Comment by lacabra** [comment]
@ainsleys: let's leave issues like this open. Only close the ones that are resolved. Otherwise: out of sight, out of mind :)
